lib/pact_broker/client/pacts.rb

```diff
@@ -21,6 +21,11 @@ def publish options
         end
       end
 
```

Comment:
Thanks. Yes, I think it would be nicer to pass in the names instead of the json. The content of the pact is irrelevant.